src/main/java/de/medizininformatikinitiative/fhir_data_evaluator/FhirDataEvaluatorApplication.java

```diff
@@ -172,6 +196,52 @@ private String getMeasureFile() {
         return readMeasure;
     }
 
+    private String readFromInputStream(InputStream inputStream)
+            throws IOException {
+        StringBuilder resultStringBuilder = new StringBuilder();
+        try (BufferedReader br
+                     = new BufferedReader(new InputStreamReader(inputStream))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                resultStringBuilder.append(line).append("\n");
+            }
+        }
+        return resultStringBuilder.toString();
+    }
+
+    private String getBundleTemplate() {
+        try {
+            return readFromInputStream(EvaluationExecutor.class.getResourceAsStream(TRANSACTION_BUNDLE_TEMPLATE_FILE));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private String createTransactionBundle(String date, String report) {
```

Comment:
@bastianschaffer when sending the document reference and measure bundle to the server:
the DocRef should be update if it already exists for the same project identifier
and the MeasureReport should always be created new.
The DocRef should then always link to the newest MeasureReport.

=> DocRefs are updated and overwritten
=> MeasureReports are created new and old ones are kept